SpatialGDK/Source/SpatialGDKServices/Private/SpatialGDKServicesModule.cpp

```diff
@@ -2,6 +2,16 @@
 
 #include "SpatialGDKServicesModule.h"
 
+#include "Editor/WorkspaceMenuStructure/Public/WorkspaceMenuStructure.h"
```

Comment:
That's a lot of headers :) Do we need all of them in SpatialGDKServicesModule.cpp? Or we can move some of them to SSpatialOutputLog.cpp?

Reply:
Yes some of these can now be removed due to refactoring. Will get on it, cheers!